src/pages/all-pokemons.tsx

```diff
@@ -72,7 +73,10 @@ export default function AllPokemonsPage() {
             </TableRow>
           ) : (
             pagePokemons.map((pokemon) => (
```

Comment:
```suggestion
            pagePokemons.map((pokemon: Pokemon) => (
```

and remove as pokemon

Reply:
Done.